http-add-on/templates/operator/service.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   ports:
   - name: https
     port: {{ default 8443 .Values.operator.port }}
-    targetPort: https
+    targetPort: metrics
```

Comment:
Care to elaborate on this?

Reply:
Sure, there isn't any port named `https` in the operator:
https://github.com/kedacore/charts/blob/1931630189a5b57e5114e407d5b5be4354c27af0/http-add-on/templates/operator/deployment.yaml#L54-L58

And there is one occurrence where `.Value.operator.port` is used:
https://github.com/kedacore/charts/blob/main/http-add-on/templates/operator/deployment.yaml#L26

That's why I supposed that the target port is the metrics port